admin/tabs/server/logViewer/LogViewerModel.js

```diff
@@ -103,6 +112,26 @@ export class LogViewerModel extends HoistModel {
         }
     }
 
+    async deleteSelectedAsync() {
+        try {
+            const recs = this.filesGridModel.selectedRecords,
+                count = recs.length;
+            if (!count ||
+                !(await XH.confirm({title: 'Confirm Delete?', message: `Delete ${count} files?`}))) {
```

Comment:
I haven't seen this pattern of using await within the condition before - cool!

Reply:
Although I must admit I find it a bit hard to parse, given that it may or may not be async depending on the previous clause. Might be clearer to split them up into two different checks, one for the early out and one for async user confirmation?